src/update.py

```diff
@@ -17,21 +20,21 @@ def download_and_index_item(item_id):
     url = "https://hacker-news.firebaseio.com/v0/item/%s.json?print=pretty" % item_id
     h = {'User-Agent': "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.124 Safari/537.36"}
     response = yield http_client.fetch(url, headers=h)
-    item = json.loads(response.body)
+    item = json.loads(response.body.decode())
```

Comment:
I think this needs to be `decode('utf-8')` or otherwise it'll fail for eg id=8837508 (failed for me)